homeassistant/components/systemmonitor/util.py

```diff
@@ -25,6 +25,11 @@ def get_all_disk_mounts() -> set[str]:
                 "No permission for running user to access %s", part.mountpoint
             )
             continue
+        except OSError as err:
+            _LOGGER.warning(
```

Comment:
```suggestion
            _LOGGER.debug(
```

Ideally we would warn if they have the entity enabled, but in this case they probably don't so it will probably generate issues which will have no solution

Reply:
Probably makes sense. I put it as warning to not having to get questions why mountpoint x is missing but you're likely right I would get more others instead.